benchmark/index.js

```diff
@@ -263,15 +263,11 @@ test('load two indexes concurrently', (t) => {
   const bob = ssbKeys.loadOrCreateSync(path.join(dir, 'secret-b'))
 
   function waitForFile(filename, cb) {
-    fs.exists(filename, function(exists) {
-      if (exists) cb()
-      else setTimeout(() => waitForFile(filename, cb), 250)
-    })
+    if (fs.existsSync(filename)) cb()
+    else setTimeout(waitForFile, 250, filename, cb)
   }
 
   db.onReady(() => {
-    const done = multicb({ pluck: 1 })
-    const start = Date.now()
```

Comment:
We shouldn't trigger the `start` before the `waitForFile` is done, because it uses some 250ms timers.